src/bitcoin-cli.cpp

```diff
@@ -404,7 +404,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         return UNKNOWN_NETWORK;
     }
     uint8_t m_details_level{0}; //!< Optional user-supplied arg to set dashboard details level
-    bool DetailsRequested() const { return m_details_level > 0 && m_details_level < 5; }
```

Comment:
> The upper bound check is no longer needed after the merge of PR 21192.

Nit, looks like the bound check is for upper and lower bounds (above zero and bellow 5).

Reply:
Thank you for taking a look! `m_details_level` is capped at a maximum value of 4 since https://github.com/bitcoin/bitcoin/pull/21192/files; that line is currently `m_details_level = std::min(n, NETINFO_MAX_LEVEL);`

Reply:
I think you misunderstood my comment (or I am not understanding yours).  min provides both an upper an lower bound here, so it's just bounded.  In the commit you say "upper bound" but I'm saying it should just read "bound".

Reply:
Oh ok, updated the commit message per s/upper bound/bound/ as you suggested.